### PR TITLE
APAM-87 Support Swift Package Manager

### PR DESCRIPTION
**Pull request by weiping-awx**
Support Swift Package Manager

### CODE REVIEW

File: Airwallex/Core/include/AWXAPIClient.h [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/weiping.li/awx/airwallex-payment-ios/Airwallex/Core/Sources/AWXAPIClient.h
```
Comment (aw-hector) [inline Airwallex/Core/include/AWXAPIClient.h:1]: Could symbolic link refers to a relative path?

File: Airwallex/Redirect/AWXRedirectActionProvider.h [hunk 2]
```diff
@@ -15,6 +15,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface AWXRedirectActionProvider : AWXDefaultActionProvider
 
+- (void)confirmPaymentIntentWithPaymentMethodName:(NSString *)paymentMethodName
+                                   additionalInfo:(NSDictionary<NSString *, NSString *> *_Nullable)additionalInfo NS_SWIFT_NAME(confirmPaymentIntent(with:additionalInfo:));
+
+- (void)confirmPaymentIntentWithPaymentMethodName:(NSString *)paymentMethodName
+                                   additionalInfo:(NSDictionary<NSString *, NSString *> *_Nullable)additionalInfo
+                                             flow:(AWXPaymentMethodFlow)flow NS_SWIFT_NAME(confirmPaymentIntent(with:additionalInfo:flow:));
```
Comment (aw-hector) [inline Airwallex/Redirect/AWXRedirectActionProvider.h:23]: Should additionalInfo = nil also be treated as another OC method?

File: Airwallex/Redirect/AWXRedirectActionProvider.m [hunk 3]
```diff
@@ -45,6 +45,19 @@ - (void)handleNextAction:(AWXConfirmPaymentNextAction *)nextAction {
...
+    [self confirmPaymentIntentWithPaymentMethodName:paymentMethodName additionalInfo:additionalInfo flow:AWXPaymentMethodFlowApp];
+}
+
+- (void)confirmPaymentIntentWithPaymentMethodName:(NSString *)paymentMethodName
+                                   additionalInfo:(NSDictionary<NSString *, NSString *> *_Nullable)additionalInfo
+                                             flow:(AWXPaymentMethodFlow)flow {
+    AWXPaymentMethod *paymentMethod = [[AWXPaymentMethod alloc] init];
+    paymentMethod.type = paymentMethodName;
+    paymentMethod.additionalParams = additionalInfo;
+    [self confirmPaymentIntentWithPaymentMethod:paymentMethod paymentConsent:nil flow:flow];
```
Comment (aw-hector) [inline Airwallex/Redirect/AWXRedirectActionProvider.m:58]: Has this method been OCMVeirfy-ed to see whether it has been executed with right parameters?

File: Frameworks/WechatOpenSDK.xcframework/Info.plist [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		<dict>
+			<key>HeadersPath</key>
+			<string>Headers</string>
+			<key>LibraryIdentifier</key>
+			<string>ios-arm64_i386_x86_64-simulator</string>
+			<key>LibraryPath</key>
+			<string>libWechatOpenSDK.a</string>
+			<key>SupportedArchitectures</key>
+			<array>
+				<string>arm64</string>
```
Comment (aw-hector) [inline Frameworks/WechatOpenSDK.xcframework/Info.plist:31]: Looks like arm64 simulator is now supported, maybe we shouldn't EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64 anymore, better to verify on simulator though.

File: README.md [hunk 5]
```diff
@@ -64,6 +64,23 @@ Run the following command:
...
+Use the following URL for the Airwallex package:
+https://github.com/airwallex/airwallex-payment-ios
+3. Version Requirement
+Ensure you specify version 5.7.0 or later.
+
+#### Components Available for Integration
+You can add `Airwallex` to include all components, or selectively add the following components to your project, depending on your payment needs:
+
+- `AirwallexApplePay`: For integrating Apple Pay.    
+- `AirwallexCard`: For Card Pay services.
```
Comment (aw-hector) [inline README.md:81]: `For card payments` might be better

File: README.md [hunk 6]
```diff
@@ -64,6 +64,23 @@ Run the following command:
...
+- `AirwallexRedirect`: To support payments via schema redirection.
```
Comment (aw-hector) [inline README.md:82]: Schema redirection might be confusing, can be url/deeplink redirection.

**Comment by weiping-awx** [comment]
Comments addressed @aw-hector

**Review by aw-hector — APPROVED** [review]
Awesome work! 👏

**Pull request merged**